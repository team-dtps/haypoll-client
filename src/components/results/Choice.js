import React from 'react';
import PropTypes from 'prop-types';

function Choice({ total, choice }) {
  return (
    <li>
      {choice} = {total}%
    </li>
  );
}

Choice.propTypes = {
  total: PropTypes.number.isRequired,
  choice: PropTypes.string.isRequired
};

export default Choice;
