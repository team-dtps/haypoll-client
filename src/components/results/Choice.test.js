import React from 'react';
import renderer from 'react-test-renderer';
import Choice from './Choice';

describe('Choice snapshot', () => {
  it('can match a snapshot', () => {
    const props = {
      choice: 'hay',
      total: 50
    };
    const tree = renderer.create(
      <Choice total={props.total} choice={props.choice}/>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
