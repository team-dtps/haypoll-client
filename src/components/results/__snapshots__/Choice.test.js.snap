// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Choice snapshot can match a snapshot 1`] = `
<li>
  hay
   = 
  50
  %
</li>
`;
